### DISCUSSION TITLE
Can't install Camera Settings Control (17) on K1-Max

### DISCUSSION THREAD

**Discussion by Y0Sam** [Guilouz/Creality-Helper-Script-Wiki#676] [Issues]
I have A K1-Max for a couple of months now and Have used the Helper-Script to install stuff which works fine.
When trying to install the Camera Settings Control (17) feature i get the  ✗ This is not compatible with the new hardware version of the camera! message.

When will this new camera be supported?

system info:
System: Linux (Kernel 4.4.94)
Firmware: 1.3.3.29
Hostname: K1Max-3124
Device SN: 29422328053124

**Comment by Digitalashley** [comment]
i have the same response when trying to install camera controls. is there anything that can make i work as of now? hope support will be added if its not possible atm

**Comment by JohnChristianCastillo** [comment]
How I solved it:
I just finished setting up my k1-Max and encountered this post
Fluidd does not really need the Camera Settings Control (at least at this point in time) if you only want the live feed.
Just go to: Settings->Cameras->Add Camera   then add a name to the camera and click add
now the camera appears in the Home tab

**Comment by Lampies69** [comment]
High guys do 18 first and then 17, that worked for me.
